runtime/rococo/src/lib.rs

```diff
@@ -98,7 +92,7 @@ pub const VERSION: RuntimeVersion = RuntimeVersion {
 	spec_name: create_runtime_str!("rococo"),
 	impl_name: create_runtime_str!("parity-rococo-v2.0"),
 	authoring_version: 0,
-	spec_version: 9170,
+	spec_version: 9180,
```

Comment:
I think we shouldn't be fiddling with this value unless we're making a release, but I'm not a release engineer so I'm not sure about this.

Reply:
hi i'm releng
this one is fine, it's already 9180 in the release branch - syncing that to master doesn't hurt